.github/workflows/create-release-pr.yml

```
@@ -172,11 +201,7 @@ jobs:
           if [ -z "${EXISTING}" ]; then
             BODY="## Release v${NEXT_VERSION}
 
-          This PR bumps the version to \`${NEXT_VERSION}\` and will trigger an npm publish when merged.
-
-          **Before merging:**
-          - [ ] Update CHANGELOG.md with changes for this release
-          - [ ] Verify all CI checks pass"
+This PR bumps the version to \$`${NEXT_VERSION}\$` and will trigger an npm publish when merged."
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)